tests/ops/task/test_graph_task.py

```diff
@@ -477,17 +477,17 @@ def test_mongo_dry_run_queries(db) -> None:
 
     assert (
         env[CollectionAddress("postgres", "customer")]
-        == "db.postgres.customer.find({'email': ?}, {'id': 1, 'name': 1, 'email': 1, 'address_id': 1})"
+        == "db.postgres.customer.find({'email': ?}, {'address_id': 1, 'email': 1, 'id': 1, 'name': 1})"
```

Comment:
Similar re-order here, this changed but it's the same order locally and in the Github pipeline